website/src/App.tsx

```diff
@@ -35,7 +35,7 @@ export const App = observer(class App extends React.Component {
 
     if (!AppState.info) {
       return (    
-        <Loading />
+        <Loading message="Page"/>
```

Comment:
I'd rather name it `target`, or `page` or something, as we're not showing a message per se, but the place we are loading.

Reply:
I would undo the change.
I don't think the information is necessary, is it?

Reply:
We can easily live without it ... :innocent: